docs/install.rst

```diff
@@ -23,7 +23,7 @@ Start by getting the source::
 You need to create a virtual environment for Python libraries. Skip the first instruction if you already have virtualenv installed::
 
     $ pip install virtualenv                       # installs virtualenv, skip if already have it
-    $ virtualenv -p python2.7 venv                 # create a virtual env in the folder `venv`
+    $ virtualenv -p 2.7 venv                       # create a virtual env in the folder `venv`
```

Comment:
Did you test this on Mac and Windows? The docs seems to still suggest the current syntax ~ https://virtualenv.pypa.io/en/stable/reference/#cmdoption-p